source/elements/oneTBB/source/algorithms/functions/collaborative_call_once_func.rst

```diff
@@ -0,0 +1,85 @@
+.. SPDX-FileCopyrightText: 2019-2021 Intel Corporation
```

Comment:
Since you added the new file the copyright notice should be SPDX-FileCopyrightText: 2021 Intel Corporation
That is - no range of years, only current one.